lib/std/target.zig

```diff
@@ -8,6 +8,7 @@ pub const Target = struct {
     os: Os,
     abi: Abi,
     ofmt: ObjectFormat,
+    page_size: ?usize = null,
```

Comment:
I don't think page size should be part of the target since it can be determined at runtime.

Reply:
We've got a problem then since there's some parts of std which depend on comptime page size for creating arrays.

Reply:
OK, let's solve the problem